rust/src/bls/api.rs

```diff
@@ -175,6 +175,58 @@ pub unsafe extern "C" fn fil_verify(
     verify_sig(&signature, digests.as_slice(), public_keys.as_slice()) as libc::c_int
 }
 
+/// Verify that a signature is the aggregated signature of the hhashed messages
+///
+/// # Arguments
+///
+/// * `signature_ptr`             - pointer to a signature byte array (SIGNATURE_BYTES long)
+/// * `messages_ptr`              - pointer to an array containing the pointers to the messages
+/// * `messages_sizes_ptr`        - pointer to an array containing the lengths of the messages
+/// * `messages_len`              - length of the two messages arrays
+/// * `flattened_public_keys_ptr` - pointer to a byte array containing public keys
```

Comment:
@dignifiedquire - nitpick: you're missing a comment for `flattened_public_keys_len` parameter